src/treeinterp/JsonReader.cpp

```diff
@@ -24,9 +24,13 @@ std::string JsonReader::getFilePath(std::string name) {
         }
         smallPath.append(word + "/");
     }
-    //std::cout << "Smallpath: " << smallPath << std::endl;
+
+    std::cout << "looking for jsons at smallpath: " << smallPath << std::endl;
     // should be at /home/[user]/roboteamtwente/workspace/src/roboteam_ai/ ish right now
-    smallPath.append("roboteam_ai/src/jsons/" + name + ".json");
+
+    smallPath.append("../../roboteam_ai/src/jsons/" + name + ".json");
```

Comment:
This is a bit problematic, shouldn't have to backtrack the path but find a way to do it immediately.